packages/connect-voting-disputable/src/helpers/numbers.ts

```diff
@@ -0,0 +1,27 @@
+import { BigNumber, utils } from 'ethers'
```

Comment:
I wonder if it would be worth exposing `TokenAmount`s from app connectors' API (cc @bpierre).

(Also see next comment)